### PR TITLE
Update issue template to help users frame issues concisely

### PR DESCRIPTION
**Pull request by mseeley**
We receive a ~steady drip of issues in this repository for other Plex players and sometimes Plex Media Server.

Let's update the issue template to help issue submitters identify if their issue is appropriate for this repository. Let's also ask our issue submitters to help us better understand and reproduce their observations.

Look good? See any thing we can improve?

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+
+Each filed issue must please:
+
+  - include a concise description of the issue and steps to reproduce.
+  - include a sample media file or media file information when applicable.
+    - Generating samples: https://support.plex.tv/articles/201035968-generating-sample-files-from-media/.
+    - Retrieving XML media information: https://support.plex.tv/articles/201998867-investigate-media-information-and-formats/. Please attach within zip.
+  - be accompanied by _clean_ logs.
+    - PMP logs can be found here: https://support.plex.tv/articles/207338748-plex-media-player-logs/
+    - Clean log instructions: Clear your logs; reproduce the issue; attach logs in a .zip file.
```
Comment (gbooker) [inline .github/issue_template.md]: Perhaps add instructions to remove the above before submitting or put it in a comment so it doesn't show up in the resulting GHI?

File: .github/issue_template.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 
 ### Test environment
+
 **PMS Version:**
 **PMP Version:**
 **Platform (Windows/OSX/Embedded):**
```
Comment (gbooker) [inline .github/issue_template.md]: Do we want to specify other platforms?  Perhaps: `(Windows/OSX/Embedded Intel/Embedded RPi/Linux AppImage)`?

Comment (mseeley) [reply .github/issue_template.md]: I'll add in the the embedded distinctions. I'm hesitant to include app image. Issue reports from an app image maintainer is totally okay with me although I'm concerned we'd currently have challenges supporting app image installations.

File: .github/issue_template.md (hunk 1 above)
Comment (mseeley) [reply .github/issue_template.md]: There's one above but I'm happy to put one below too / make the instructions easier to yank.

**Pull request merged**